config/config.go

```diff
@@ -316,6 +317,7 @@ var defaultConf = Config{
 	Binlog: Binlog{
 		WriteTimeout: "15s",
 	},
+	BigChunkThreshold: 512,
```

Comment:
Previous, it's 1024?

Reply:
no~ this is a new added configure. in previous, initChunkSize is 1024, but now initChunkSize is 32 if stats info rows < BigChunkThreshold, else keep 1024